### ISSUE TITLE
Errors shown when linting with check-droid

### ISSUE THREAD

**Issue by dae** [ankidroid/Anki-Android-Backend#248]
The lint appears to (at least partially) still work.

**Comment by mikehardy** [comment]
Yep - the issue appears to be with some sort of class availability assumption violated by timber's lint rules

> Task :rsdroid:lint
Could not load custom lint check jar file /Users/mike/.gradle/caches/transforms-3/40bf2c11c8f0fb09cac3b5552b0978fc/transformed/timber-5.0.1/jars/lint.jar
java.lang.NoClassDefFoundError: com/android/tools/lint/client/api/Vendor

**Comment by mikehardy** [comment]
This will be resolved whenever gradle is finally updated - gradle 7 is the minimum compatible version for Timber v5+ lint rules https://github.com/JakeWharton/timber/issues/436#issuecomment-897654042

Related work that will resolve is in #171
